examples/platforms/da15000/gnu-cortex-m.ld

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Is this file still used?  If not, please remove. 
It looks like da15100.ld contains entire linker script in one file.